site/src/App/DocNavigation/DocProps.tsx

```diff
@@ -176,6 +176,10 @@ export const DocProps = () => {
   const { docsName = '', docs } = useContext(DocsContext);
   const { sourceUrlPrefix } = useConfig();
 
+  // This is temporary while we find the source of the missing props page issue
+  // eslint-disable-next-line no-console
+  console.table({ docsName, docs });
```

Comment:
This will only fire if `DocProps` is actually rendered. If the issue is at a routing level, maybe it's worth adding logs higher up, just to avoid subsequent PRs to add more logs 😅

Reply:
"DocProps wasn't rendered" is also something I'd like to know, and this will tell us that too. If it's a routing issue, no log at all. If it's the null thing, log with unexpected values.